.gitignore

```diff
@@ -1 +1,2 @@
 node_modules/
+tmp/
```

Comment:
Since the tests aren't under mocha or any test framework there's no guaranteed way to cleanup the directory without wrapping the whole file in a try/catch/finally. If any test fails then the cleanup will not happen.